examples/05_joining_tables_with_FuzzyJoin.py

```diff
@@ -358,3 +361,98 @@ def print_worst_matches(joined_table, n=5):
 #
 # Now up to you, try improving our model by adding information into it and
 # beating our result!
+
+#######################################################################
+# Using the |fa| to fuzzy join multiple tables
+# --------------------------------------------
+# A faster way to merge different tables from the World Bank
+# to `X` is to use the |fa|.
+#
+# The |fa| is a transformer that can easily chain joins of tables on
+# a main table.
+
+#######################################################################
+# Instantiating the transformer
+# ............................
+# To do this, we will first gather the auxilliary tables into a
+# list of (tables, keys):
+
+aux_wb_tables = [
+    (gdppc, "Country Name"),
+    (life_exp, "Country Name"),
+    (legal_rights, "Country Name"),
+]
```

Comment:
I would prefer if no intermediate variable aux_wb_tables was created and this was directly passed in the instantiation of the FeatureAugmenter, to make it easier to the reader to visualize what the argument type is.